src/features/game/expansion/placeable/lib/collisionDetection.ts

```diff
@@ -345,163 +343,85 @@ export function isWithinAOE(
   AOEItem: Position,
   effectItem: Position
 ): boolean {
-  const AOEItemCoordinates: Coordinates = { x: AOEItem.x, y: AOEItem.y };
+  const { x, y, height, width } = AOEItem;
 
-  const AOEItemDimensions: Dimensions = {
-    height: AOEItem.height,
-    width: AOEItem.width,
+  const isWithinRectangle = (
+    topLeft: Position,
+    bottomRight: Position
+  ): boolean => {
+    return (
+      effectItem.x >= topLeft.x &&
+      effectItem.x <= bottomRight.x &&
```

Comment:
I expcted to see < for both bottom right tests.  Is the check "too large" here or are the rectangles expressed such that `bottomRight.x-topLeft.x` is not the width?